test/spec/modules/realTimeDataModule_spec.js

```diff
@@ -58,33 +60,87 @@ const conf = {
 };
 
 describe('Real time module', function () {
-  before(function () {
-    rtdModule.attachRealTimeDataProvider(validSM);
-    rtdModule.attachRealTimeDataProvider(invalidSM);
-    rtdModule.attachRealTimeDataProvider(failureSM);
-    rtdModule.attachRealTimeDataProvider(nonConfSM);
-    rtdModule.attachRealTimeDataProvider(validSMWait);
-  });
+  let eventHandlers;
+  let sandbox;
 
-  after(function () {
-    config.resetConfig();
-  });
+  function mockEmitEvent(event, ...args) {
+    (eventHandlers[event] || []).forEach((h) => h(...args));
+  }
 
-  beforeEach(function () {
-    config.setConfig(conf);
+  before(() => {
+    eventHandlers = {};
+    sandbox = sinon.sandbox.create();
+    sandbox.stub(events, 'on').callsFake((event, handler) => {
+      if (!eventHandlers.hasOwnProperty(event)) {
+        eventHandlers[event] = [];
+      }
+      eventHandlers[event].push(handler);
+    });
   });
 
-  it('should use only valid modules', function () {
-    rtdModule.init(config);
-    expect(rtdModule.subModules).to.eql([validSMWait, validSM]);
+  after(() => {
+    sandbox.restore();
   });
 
-  it('should be able to modify bid request', function (done) {
-    rtdModule.setBidRequestsData(() => {
-      assert(getBidRequestDataSpy.calledTwice);
-      assert(getBidRequestDataSpy.calledWith({bidRequest: {}}));
+  describe('', () => {
```

Comment:
Any name for the describe block? Usually it helps to have a meaningful name especially when the test fails.

Reply:
I tried, I couldn't find one that's better than no name. These are the previously existing tests that I grouped together because I don't need the same setup for my new ones. The only name I can think that makes sense is something like "old tests" which would not help at all on failure.

Reply:
That's fine. Didn't know you clubbed old tests inside this describe block.